### ISSUE TITLE
Consider support keymap like :Q :w .etc

### ISSUE THREAD

**Issue by TheBB** [emacs-evil/evil#689]
Originally reported by: **yan zhu (Bitbucket: [halfcrazy](https://bitbucket.org/halfcrazy), GitHub: [halfcrazy](https://github.com/halfcrazy))**

----------------------------------------

Sometimes when we want to close the window or quit, we may first call Colon ':' and we need to press shift and then the "q", but we always make stupid mistake that we typed a upper case "Q" it makes me feel bad.

TLDR;

I want :Q = :q

----------------------------------------
- Bitbucket: https://bitbucket.org/lyro/evil/issue/689
